cmd/ethrex_l2/src/commands/stack.rs

```diff
@@ -277,6 +294,62 @@ impl Command {
                     current_block += U256::one();
                 }
             }
+            Command::Reconstruct {
+                genesis,
+                blobs_dir,
+                store_path,
+                coinbase,
+            } => {
+                let store = Store::new_from_genesis(
+                    store_path.to_str().expect("Invalid store path"),
+                    EngineType::Libmdbx,
+                    genesis.to_str().expect("Invalid genesis path"),
+                )?;
+
+                let genesis_header = store.get_block_header(0)?.expect("Genesis block not found");
+                let genesis_block_hash = genesis_header.compute_block_hash();
+
+                let mut new_trie = store
+                    .state_trie(genesis_block_hash)?
+                    .expect("Cannot open state trie");
+                let mut last_number = 0;
+                let mut last_hash = genesis_block_hash;
+
+                let files: Vec<std::fs::DirEntry> = read_dir(blobs_dir)?.try_collect()?;
+                for file in files.into_iter().sorted_by_key(|f| f.file_name()) {
+                    let blob = std::fs::read(file.path())?;
+
+                    if blob.len() != BYTES_PER_BLOB {
+                        panic!("Invalid blob size");
+                    }
+
+                    let blob = bytes_from_blob(blob.into());
+                    let state_diff = StateDiff::decode(&blob)?;
+                    let account_updates = state_diff.to_account_updates(&new_trie)?;
+
+                    new_trie = store
+                        .apply_account_updates_from_trie(new_trie, &account_updates)
+                        .unwrap();
```

Comment:
Shouldn't this be an `expect`?

Reply:
Yes. [462fb35](https://github.com/lambdaclass/ethrex/pull/2204/commits/462fb35fd2697b850d87c92b23b9b66ecf0d727f)